src/lib/features/feature-toggle/tests/feature-toggles.e2e.test.ts

```diff
@@ -3544,3 +3547,71 @@ test("Should not be able to create flag with a type that doesn't exist", async (
         })
         .expect(400);
 });
+
+test('GET feature returns release plans with safeguards', async () => {
```

Comment:
We were missing e2e for release plans also. So adding release plans together with safeguards.

Reply:
I am pretty sure I added this type of test in the last month. Have to find it

Reply:
ok found it. I tested release plans in milestone-progression.e2e.test.ts in enterprise repo. I prefer to test enterprise features in enterprise repo as it's super easy to setup since all the infra is already there and we don't need to write raw inserts